File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.py
class Solution(object):
    def leftRightDifference(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        result = []

        for i in range(len(nums)):
            leftsum = 0
            rightsum = 0
            for j in range(i):
                leftsum += nums[j]
            for j in range(i+1, len(nums)):
                rightsum += nums[j]
            result.append(abs(leftsum - rightsum))
            
        return result

File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var leftRightDifference = function(nums) {
    let result = [];

    for (let i = 0; i < nums.length; i++) {
        let leftSum = 0;
        let rightSum = 0;
        for (let j = 0; j < i; j++) {
            leftSum += nums[j];
        }
        for (let j = i + 1; j < nums.length; j++) {
            rightSum += nums[j];
        }
        result.push(Math.abs(leftSum - rightSum));
    }

    return result;
};